package.json

```diff
@@ -4,16 +4,17 @@
   "description": "Composable simulations for testing, development, and application previews",
   "private": true,
   "scripts": {
-    "build": "tsc -b ./tsconfig.packages.json && cd packages/ui && npm run build",
```

Comment:
this build is that points to the `./tsconfig.packages.json` which builds all typescript packages defined in that file.  We might want to keep that and put in a different script like `build:tsc` or something

Reply:
👍🏻 Sounds good. Once thing that is nice about the global build script is that because it uses `tsc -b` it's really super fast even though it's invoking three separate builds, the typescript builds use the cache after the first one.

Reply:
>  the typescript builds use the cache after the first one.

Yes, that is what those `*.tsbuildinfo` files are.  I've had things go out of sync a number of times.

`tsc -b ./tsconfig.packages.json --clean` will remove the from the whole solution